CleverPush/Source/CleverPushInstance.m

```diff
@@ -3678,7 +3678,7 @@ - (void)showAppBanner:(NSString*)bannerId notificationId:(NSString*)notification
 - (void)showAppBanner:(NSString*)bannerId channelId:(NSString*)channelId notificationId:(NSString*)notificationId {
     BOOL fromNotification = notificationId != nil;
     [CPAppBannerModule initBannersWithChannel:channelId showDrafts:isShowDraft fromNotification:fromNotification];
-    [CPAppBannerModule showBanner:channelId bannerId:bannerId notificationId:notificationId force:true];
+    [CPAppBannerModule showBanner:channelId bannerId:bannerId notificationId:notificationId force:NO];
```

Comment:
Why do we have NO here and above in L3675 the default is `force:YES`? This seems odd. I see that it seems to be similar on Android, but that should not have different values.

Reply:
1. L3675 is used for the below function. 
      `[CleverPush showAppBanner:@""];`
       (Like forcefully displaying banners, we are not checking any conditions like frequency, status, targeting, etc. 
          TargetingWithEvent, or time while using the showAppBanner function.)   

2. L3681 is used for displaying the app banner from the notification. 
     https://github.com/cleverpush/cleverpush-ios- 
     sdk/blob/0061216c53cdc4fbe5fe9102957dead93669ee66/CleverPush/Source/CleverPushInstance.m#L1757


The issue was that the customer was saying that while sending the test push from the banners, the targeting functionality was not working.